keps/sig-release/3344-release-team-roster/README.md

```diff
@@ -0,0 +1,895 @@
+
+Presently each team is selected by nomination of established community members (typically release team members in the previous release) and a shadow survey, depending on the role within the team.
+The community has identified a number of issues that arise from this procedure:
+
+- Having served on a release team is no guarantee of selection in the next team.
+  There are many cases of contributors serving for one or multiple releases, and then simply not being selected for future releases.
+  There is little feedback, and little the contributor can do other than re-apply next time.
+  This process can be stressful for contributors.
+- The release team is not "stable", as continuing requires constant re-application through the shadow survey.
+  This inhibits the ability of contributors to gain comfort and skill in a particular role. 
+- Wider community members interact with a constantly changing team of people for every release, which makes it harder for them to build working relationships with the release team.
+- With a few notable exceptions, once a contributor ends their time on the release team they don't come back.
+  This "up or out" process makes it hard to do release team work as a long-standing community role.
+
+There are a number of qualities to the current process that are desirable, and should be kept:
+
+- The shadow survey is an excellent "on ramp" into the community, providing contributors with no experience a path to [org membership](https://github.com/kubernetes/community/blob/master/community-membership.md) and it is expected that contributors may only join the release team for a release or two before moving to other parts of the community.
+- The release team having some degree of 'churn' is natural and expected.
+  An entirely static, or effectively static, team is not desirable. 
+
+Balancing these desires, this KEP introduces a long-lived "roster" of release team members.
+This roster is used to assemble release teams with recurring contributors in addition to the shadow survey for new contributors.
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+### Goals
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+- Continue to provide an "on ramp" for new community members.
+- Provide a more streamlined process for release team shadows, including easier return into future release teams.
+- Provide a "stable" position that a member may continue to be on release teams if they wish.
+- Better handle contributors who wish to take breaks from the team, and then return. 
```

Comment:
If @kubernetes/sig-release-leads are -1 on the stable roster, then we could also refocus the KEP to:

```suggestion
- Continue to provide an "on ramp" for new community members.
- Provide a more streamlined process for release team shadows, including easier return into future release teams.
- Better handle contributors who wish to take breaks from the team, and then return. 
```